docs/metrics-analysis-prometheus/prometheus.md

```diff
@@ -0,0 +1,55 @@
+# Prometheus
+
+## Introduction
+
+Prometheus is integrated with our `gh_actions_exporter` repository,
+enabling the export of GitHub Actions metrics.
+
+Prometheus is a powerful open-source monitoring and alerting system that allows
+users to collect, store, and analyze time-series data. In this guide, we will
+explore how to effectively utilize Prometheus to analyze GitHub Actions.
+
+## Understanding Prometheus Queries
+
+The idea here is not to recreate the entire Prometheus documentation; we will
+simply discuss the key points to get you started easily without getting lost in
+the plethora of information available on the Internet. I will redirect you to
+the [documentation](https://prometheus.io/docs/introduction/overview/)
+if you want to develop deeper.
+
+To proceed, I will take a typical query and break it down, discussing other
+potentially useful information to cover.
+
+Let's examining this example query:
+
+```bash
+topk(5, sum(increase(github_actions_job_cost_count_total{repository=~"$repository", runner_type=~"$runner_type", repository_visibility=~"$repository_visibility", cloud=~"$cloud"}[$__range])) by (repository) > 0)
```

Comment:
```suggestion
topk(5, sum(increase(github_actions_job_cost_count_total{}[5m])) by (repository) > 0)
```

You can add that the labels are available for filtering if needed. and add another example with hardcoded values as `$variable`

Reply:
Done